cmd/logcli/main.go

```diff
@@ -8,7 +8,7 @@ import (
 
 var (
 	app      = kingpin.New("logcli", "A command-line for loki.")
-	addr     = app.Flag("addr", "Server address.").Default("https://log-us.grafana.net").Envar("GRAFANA_ADDR").String()
+	addr     = app.Flag("addr", "Server address.").Default("").Envar("GRAFANA_ADDR").String()
```

Comment:
I think we should check that `*addr != ""` before [it's used in query.go](https://github.com/grafana/loki/blob/30cba7ad1d90adda8158f413cb8d48842241dac2/cmd/logcli/client.go#L50).